app/models/chargeable_field.rb

```diff
@@ -23,4 +23,14 @@ def self.seed_data
     fixture_file = File.join(FIXTURE_DIR, 'chargeable_fields.yml')
     File.exist?(fixture_file) ? YAML.load_file(fixture_file) : []
   end
+
+  def self.chargeable_cols_on_metric_rollup
+    existing_cols = MetricRollup.attribute_names
+    chargeable_cols = pluck(:metric) & existing_cols
+    chargeable_cols.map! { |x| VIRTUAL_COL_USES[x] || x }
+  end
+
+  VIRTUAL_COL_USES = {
```

Comment:
We do :) 

https://github.com/ManageIQ/guides/blob/master/coding_style_and_standards.md#ruby-style-guide
->
https://github.com/bbatsov/ruby-style-guide#consistent-classes